_data/enterprise-landing-page-tabs-code.yml

```diff
@@ -0,0 +1,98 @@
+- id: eks
+  title: EKS cluster
+  content: |
+    <figure><figcaption>eks-cluster/main.tf</figcaption>
+      <pre>
+      <code class="language-hcl">
+        module "eks_cluster" {
+          source  = "github.com/gruntwork-io/service-catalog//eks-cluster"
+          version = "v0.51.0"
+
+          service_name  = "my-heroku-app"
+          cluster_name  = "my-prod-cluster"
+
+          vpc_id = dependency.vpc.outputs.vpc_id
+
+          # Simplify management by running the control plane on fargate
+          schedule_control_plane_services_on_fargate = true
```

Comment:
Enterprises probably won't want that, so I'd remove it.

Reply:
I took a pass and streamlined this module configuration block. Again, this was a relic of being WIP and copy pasting from our examples.